src/gfn/gym/helpers/box_utils.py

```diff
@@ -550,6 +568,7 @@ def forward(
             desired_out[..., 1 + self._n_comp_max :]
         )
 
+        # assert desired_out.shape == batch_shape + (1 + 5 * self.n_components,)  # TODO: check why this fails
```

Comment:
This shape check fails. Not sure if the computation is correct?

Reply:
I would need to verify on my side, but all tests complete ok on my side. I find it as bit confusing, because we test the scripts as well.

Is it possible you introduced a bug?